packages/mux-player/src/html.ts

```diff
@@ -108,13 +109,13 @@ export function processPart(part: TemplatePart, value: unknown): void {
 }
 
 const templates = new WeakMap<TemplateStringsArray, HTMLTemplateElement>();
-const renderedTemplates = new WeakMap<Node | NodeTemplatePart, HTMLTemplateElement>();
-const renderedTemplateInstances = new WeakMap<Node | NodeTemplatePart, TemplateInstance>();
+const renderedTemplates = new WeakMap<Node | ChildNodePart, HTMLTemplateElement>();
+const renderedTemplateInstances = new WeakMap<Node | ChildNodePart, TemplateInstance>();
 export class TemplateResult {
   constructor(
     public readonly strings: TemplateStringsArray,
     public readonly values: unknown[],
-    public readonly processor: TemplateTypeInit
+    public readonly processor: any
```

Comment:
question(non-blocking): Why did this have to be converted to `any`? Is it possible to avoid this?

Reply:
yes, there was always an annoying issue that I didn't want to spend time on. I'll check when moving this to MC